static/app/utils/analytics/issueAnalyticsEvents.tsx

```diff
@@ -88,4 +90,10 @@ export const issueEventMap: Record<IssueEventKey, string | null> = {
   'issue.quick_trace_status': 'Issue Quick Trace Status',
   'span_view.embedded_child.hide': 'Span View: Hide Embedded Transaction',
   'span_view.embedded_child.show': 'Span View: Show Embedded Transaction',
+
+  // Performance Issue specific events here
+  'issue_details.performance.autogrouped_siblings_toggle':
```

Comment:
reload PR?

Reply:
Yep I'll add all of these to Reload as well